source/documentation/monitoring_apps/metrics.md

```diff
@@ -2,110 +2,95 @@
 
-You can set up Prometheus to request metrics from your region's API endpoint by using [Docker](https://www.docker.com/) [external link] to run a local instance of Prometheus.
-
-1. Save the following script as `test-metrics.sh`:
+1. [Push the PaaS Prometheus exporter app](/deploying_apps.html#deploying-public-apps) to Cloud Foundry without starting the app:
 
 	```
-	#!/usr/bin/env bash
-	set -ue
-
-	echo "
-	global:
-	  scrape_interval: 1m
-	  evaluation_interval: 1m
-	  scrape_timeout: 1m
-
-	scrape_configs:
-	  - job_name: PaaS
-	    bearer_token: $(cf oauth-token | sed 's/bearer //')
-	    scheme: https
-	    static_configs:
-	      - targets:
-	        - API_ENDPOINT:443
-	" > prometheus.yml
-
-	docker run --publish 9090:9090 \
-	           --volume "$PWD/prometheus.yml:/etc/prometheus/prometheus.yml" \
-	           prom/prometheus
+	cf push --no-start prometheus-exporter --hostname prometheus-exporter-ORGNAME
 	```
-	where API_ENDPOINT is:
-	- `https://metrics.cloud.service.gov.uk/metrics` for the Ireland region
-	- `https://metrics.london.cloud.service.gov.uk/metrics` for the London region
 
-1. Make the script executable:
+	where `ORGNAME` is the name of your org. For example:
 
 	```
-	chmod +x test-metrics.sh
+	cf push --no-start prometheus-exporter --hostname prometheus-exporter-exampleorg
 	```
 
-1. Execute the script:
+	Running this command deploys the PaaS Prometheus exporter app to `https://prometheus-exporter-exampleorg.cloudapps.digital` without starting the app.
+
+	Refer to the [app names and domain hostnames documentation](/deploying_apps.html#app-names-and-domain-hostname-clash) for more information on how to avoid duplicating existing app names.
+
+1. Set the following mandatory environment variables in the PaaS Prometheus exporter app by running `cf set-env prometheus-exporter NAME VALUE`:
+
+	|Name|Value|
+	|:---|:---|
+	|`API_ENDPOINT`|- `https://api.cloud.service.gov.uk` for Ireland<br>- `https://api.london.cloud.service.gov.uk` for London|
+	|`USERNAME`|Cloud Foundry User|
+	|`PASSWORD`|Cloud Foundry Password|
+
+	You should use the `cf set-env` command for these mandatory variables as they contain secret information, and this method will keep them secure.
+
+	You can also set environment variables by changing the manifest file. You should do this for optional environment variables that do not contain secret information. Refer to the [PaaS Prometheus exporter repository](https://github.com/alphagov/paas-prometheus-exporter) [external link] for more information.
+
+1. Configure your Prometheus service's metrics endpoint. The metrics endpoint is the deployed PaaS Prometheus exporter app URL with `/metrics` added on to the end. For this example:
 
 	```
-	./test-metrics.sh
+	https://prometheus-exporter-exampleorg.cloudapps.digital/metrics
 	```
 
-	If the script executes successfully, you will see the message:
+1. Start the PaaS Prometheus exporter app:
```

Comment:
Logically a tenant should start the application first (too see that it works) and then set it up in Prometheus.